oranda-css/css/variables.css

```diff
@@ -1,10 +1,31 @@
 :root {
-    --dark-color: #141414;
-    --light-color: #ffffff;
-    --link-color: #0284c7;
+    /* Base colors for text and background */
+    --dark-fg-color: #ffffff;
+    --light-fg-color: #141414;
+    --light-bg-color: var(--dark-fg-color);
+    --dark-bg-color: var(--light-fg-color);
+    --fg-color: var(--light-fg-color);
+    --bg-color: var(--light-bg-color);
+
+    /* Link colors */
+    --light-link-color: #0284c7;
+    --dark-link-color: #8BB9FE;
+    --link-color: var(--light-link-color);
```

Comment:
any reason some of these are hardcoded here, and others are imported from themes/axo.css..?

Reply:
these colors are for the default theme (which is not our axo theme)